source/index.html.md

```diff
@@ -1411,10 +1414,11 @@ Use this endpoint to fetch status of any order
 
 ### Parameters
 
-| Name      | Required | Example                              | Description                    |
-|-----------|----------|--------------------------------------|--------------------------------|
-| id        | Yes      | ead19992-43fd-11e8-b027-bb815bcb14ed | The ID of the order            |
-| timestamp | Yes      | 1524211224                           | When was the request generated |
+| Name            | Required | Example                              | Description                      |
+|-----------------|----------|--------------------------------------|----------------------------------|
+| id              | No       | ead19992-43fd-11e8-b027-bb815bcb14ed | The ID of the order              |
+| client_order_id | No       | 8a1d1e4c-c895-11e8-9dff-df1480546936 | The Client Order ID of the order |
```

Comment:
Let's use `2022.02.14-btcinr_1` only as the example value for `client_order_id` throughout the page - uniformity helps to follow better.